internal/bundle/testdata/fail_PolicyMissingChecksRuleID.yaml

```diff
@@ -0,0 +1,209 @@
+
```

Comment:
all files should be ending with `.mql.yaml` and then formatted with `cnspec policy fmt`